resources/public/api/conf/2.0/retrieve_income_tax_crystallisation.yaml

```diff
@@ -63,11 +48,21 @@ get:
         </tr>
       </tbody>
     </table>
- 
 
   parameters:
     - $ref: './common/pathParameters.yaml#/components/parameters/nino'
-    - $ref: './common/queryParameters.yaml#/components/parameters/taxYear'
+    - name: taxYear
+      in: query
+      description: |
+        The tax year the data applies to. The earliest permitted tax year is 2017-18.
+        
+        If a tax year is not specified then all obligation from CY-4 are returned. (CY = Current Tax Year)
+        eg. if current TY is 2023-24 then all obligations from 2019-20 till 2023-34 are returned.
+      schema:
+        type: string
+        example: 2022-23
+      required: false
+    - $ref: './common/queryParameters.yaml#/components/parameters/status'
```

Comment:
The description for status query param should match the one finalised by Ben in the tech spec.

Reply:
correct